packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -202,20 +202,19 @@ module.exports = {
     ];
   },
   plugins: [
-    // Makes the public URL available as %PUBLIC_URL% in index.html, e.g.:
+    // Makes some environment variables available in index.html.
+    // The public URL is available as %PUBLIC_URL% in index.html, e.g.:
     // <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
     // In development, this will be an empty string.
-    new InterpolateHtmlPlugin({
-      PUBLIC_URL: publicUrl
-    }),
+    new InterpolateHtmlPlugin(env.vars),
```

Comment:
I noticed the CI passed even before you corrected this. Do you have time to add a test case?

However, I'm not sure if I'm fond of the `.vars` and `.string` names -- do you have a better suggestion @gaearon?